internal/services/appservice/helpers/web_app_schema.go

```diff
@@ -879,7 +879,8 @@ func windowsApplicationStackSchema() *pluginsdk.Schema {
 					Type:     pluginsdk.TypeString,
 					Optional: true,
 					ValidateFunc: validation.StringInSlice([]string{
-						"v3.0",
```

Comment:
i'm afraid we can't just remove this value, could we add it back in and conditionally remove it when the 4.0 flag is enabled? thanks